### DISCUSSION TITLE
WooCommerce Blocks Checkout Field - Add field to admin New Order screen

### DISCUSSION THREAD

**Discussion by GlennTaylorDigital** [woocommerce/woocommerce#47955] [General]
Hi all

I'm currently working on a project that I've been making use of the```__experimental_woocommerce_blocks_register_checkout_field``` function to support adding a date selection dropdown field to the checkout. 

If/when somebody pushes a new order through from the checkout I can go in to their order in the backend and amend the date they've selected by hitting the little pencil icon alongside it:

<img width="691" alt="image" src="https://github.com/woocommerce/woocommerce/assets/1357597/bde8c5b2-4e0b-44a3-8b87-b3b4bcbc875f">

This works great, but I need to add the ability for the ecommerce team to be able to set a delivery date when creating a new order from the admin screen. For an existing order, the dropdown shows when clicking the pencil icon on the Shipping Address section.

<img width="691" alt="image" src="https://github.com/woocommerce/woocommerce/assets/1357597/55604bd1-1656-4863-b819-ee01f6778ac3">

But on a new order I don't see this field.

Is there a way to make it show here, or would I need to add a select field within ```woocommerce_admin_order_data_after_order_details```

Then use the ```woocommerce_process_shop_order_meta``` action to update the _additional_fields meta data where the ```__experimental_woocommerce_blocks_register_checkout_field``` function would usually store the additional checkout fields?

Help/pointers are massively appreciated!

Thanks in advance.

**Comment by senadir** [comment]
I think you can reapply the same hooks being added to the order edit screen to that one, thou I'm not sure how that would work with no order in place.

Additionally, as a workaround, you can create the order and then edit it again for them to show.
